project3/style.css

```
@@ -0,0 +1,2586 @@
+/* ===================
+COLOR PALETTE
+===================*/
+
+/* .sr-only {
+    border: 0;
+    clip: rect(0 0 0 0);
+    overflow: hidden;
+    margin: -1px;
+    padding: 0;
+    position: absolute !important;
+    height: 1px;
+    width: 1px;
+} */
+
+/* .button-reset[disabled] {
+    cursor: default;
+} */
+
+
+:root {
+    /*  ==================
+  COLOR PALETTE
+  ==================*/
+
+    --color-red: #c61a3a;
+    --color-green: #71f79f;
+    --color-offblack: #1e2019;
+    --color-grey: #d1d5de;
+    --color-offwhite: #fafafa;
+    --color-blue: #163346;
+    /* Reminder that you can name your variables anything you want, they just have to start with two dashes (--) */
+    /* ㅇㅣ거 하면 이름대로 색 바꿀 수 있음 */
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+.button-reset {
+    background: transparent;
+    border: 0;
+    cursor: pointer;
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: inherit;
+    padding: 0;
+    text-align: inherit;
+}
+
+button[type="submit"] {
+    background-color: var(--color-green);
+    font-family: Teko;
```

This is missing the generic font family:
```suggestion
    font-family: Teko, sans-serif;
```